checks.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+res_pip_list=$(pip freeze)
+if [[ ! $(echo $res_pip_list | grep black) ]] ; then
+  pip install black
+fi
+if [[ ! $(echo $res_pip_list | grep flake8) ]] ; then
+  pip install flake8
+fi
+if [[ ! $(echo $res_pip_list | grep isort) ]] ; then
+  pip install isort
+fi
+if [[ ! $(echo $res_pip_list | grep mypy) ]] ; then
+  pip install mypy
+fi
```

Comment:
The automatic installation of the dependencies is a good idea, but I guess users may be surprised if the script does not ask users to install them. I think it would be polite if the script proceeds the installation as follows:

1. Find missing libraries
2. Ask users to install them
3. If they type `y` or `yes`, the script install them
4. Otherwise, terminate the script

What do you think of it?

Reply:
Thanks for the comment. I fixed the script to ask users to install the missing dependencies as you suggested.